src/pages/Profile/Profile.tsx

```diff
@@ -418,25 +418,47 @@ export const Profile: React.FC = () => {
                 {
                   avatarHasBeenClicked
                     ? (
-                      <Box px={1.5} pb={2}>
-                        <TextField
-                          error={(formState.touched.userAvatarURL && formState.errors.userAvatarURL) || !validImage}
-                          fullWidth
-                          helperText={
-                            ((formState.touched.userAvatarURL &&
-                          formState.errors.userAvatarURL) || !validImage) &&
-                        formState.errorMsgs.userAvatarURL
-                          }
-                          label={t("profileTab.overviewSubTab.userRelatedLabels.userAvatarURL")}
-                          margin='dense'
-                          name='userAvatarURL'
-                          onChange={handleChange}
-                          onFocus={handleFocus}
-                          type='url'
-                          value={formState.values.userAvatarURL}
-                          variant='outlined'
-                        />
-                      </Box>
+                      <>
+                        <Box px={1.5} pb={2}>
+                          <TextField
+                            error={(formState.touched.userAvatarURL && formState.errors.userAvatarURL) || !validImage}
+                            fullWidth
+                            helperText={
+                              ((formState.touched.userAvatarURL &&
+                                formState.errors.userAvatarURL) || !validImage) &&
+                              formState.errorMsgs.userAvatarURL
+                            }
+                            label={t("profileTab.overviewSubTab.userRelatedLabels.userAvatarURL")}
+                            margin='dense'
+                            name='userAvatarURL'
+                            onChange={handleChange}
+                            onFocus={handleFocus}
+                            type='url'
+                            value={formState.values.userAvatarURL}
+                            variant='outlined'
+                          />
+                        </Box>
+
+                        {
+                          ssoIsActive &&
```

Comment:
Is there any design for this? Or was this confirmed with design team?

Reply:
This was confirmed with the design team, as described in the corresponding features' ticket.